### DISCUSSION TITLE
zh-CN Traditional Chinese

### DISCUSSION THREAD

**Discussion by zhhl9101** [openai/whisper#777] [Q&A] [answered]
My script is as below:

model = whisper.load_model("base")
#reco = model.transcribe(audio)
reco = model.transcribe(audio, language='zh')

Then, i got reco like that: "暫停語言模式", it is Traditional Chinese, 
What i want is "暂停语言模式", Simplified Chinese，
So what should i do ?

**Comment by jongwook** [comment, marked as answer]
See https://github.com/openai/whisper/discussions/277#discussioncomment-3832154
